app/src/common/shared/org/mozilla/vrbrowser/browser/SettingsStore.java

```diff
@@ -738,5 +736,15 @@ public void setDownloadsSortingOrder(@SortingContextMenuWidget.Order int order)
     public @Storage int getDownloadsSortingOrder() {
         return mPrefs.getInt(mContext.getString(R.string.settings_key_downloads_sorting_order), DOWNLOADS_SORTING_ORDER_DEFAULT);
     }
+
+    public void setFxAWebChannelsEnabled(boolean isEnabled) {
+        SharedPreferences.Editor editor = mPrefs.edit();
+        editor.putBoolean(mContext.getString(R.string.settings_key_webchannels_enabled), isEnabled);
+        editor.commit();
+    }
+
+    public boolean isFxAWebChannelsEnabled() {
+        return mPrefs.getBoolean(mContext.getString(R.string.settings_key_webchannels_enabled), FXA_WEBCHANNELS_ENABLED);
+    }
```

Comment:
Is there a reason we want to be able to turn WebChannels off? For QA and testing?

Reply:
I just left it just in case we want to enable/disable easily in the future. I can remove it, is not really used right now.

Reply:
I wasn't sure if there was a reason. I would remove it if we don't want to support the previous version.

Reply:
Removed